styx-api-service/src/main/java/com/spotify/styx/api/BackfillResource.java

```diff
@@ -119,12 +118,12 @@ public BackfillResource(String schedulerServiceBaseUrl, Storage storage) {
     );
 
     return cat(
-        Api.prefixRoutes(entityRoutes, V1),
-        Api.prefixRoutes(routes, V1)
+        Api.prefixRoutes(entityRoutes, V2),
+        Api.prefixRoutes(routes, V2)
     );
   }
 
-  private BackfillsPayload getBackfills(RequestContext rc) {
+  public BackfillsPayload getBackfills(RequestContext rc) {
```

Comment:
The deprecated BackfillResource has no logic but calling the new version, so that's why it is public now.